x-pack/plugins/triggers_actions_ui/public/common/constants/index.ts

```diff
@@ -7,6 +7,10 @@
 
 export { COMPARATORS, builtInComparators } from './comparators';
 export { AGGREGATION_TYPES, builtInAggregationTypes } from './aggregation_types';
+export { loadAllActions, loadActionTypes } from '../../application/lib/action_connector_api';
+export { ConnectorAddModal } from '../../application/sections/action_connector_form';
+export type { ActionConnector } from '../..';
+
```

Comment:
@spong needs these exported for the security assistant implementation: https://github.com/elastic/kibana/pull/156933